src/framework/ui/select/select.component.tsx

```diff
@@ -57,6 +57,7 @@ type IconProp = (style: ImageStyle, visible: boolean) => IconElement;
 type SelectChildren = [SelectOptionsListElement, TextElement, ControlElement];
 
 export type SelectOption = Array<SelectOptionType> | SelectOptionType;
+export type KeyExtractorType = (item: SelectOptionType) => string;
```

Comment:
I guess it would be more valid if we add an index here too. Just like it is implemented in a [FlatList](https://facebook.github.io/react-native/docs/flatlist#keyextractor)

Reply:
We can't do this cause there are groups options